jenkins-master/jobs/trigger-ondemand/workspace/trigger.py

```diff
@@ -84,7 +84,7 @@ def get_test_packages_url(properties):
 
     revision = properties['revision'][:12]
 
-    client = TreeherderClient(host='treeherder.mozilla.org', protocol='https')
+    client = TreeherderClient(server_url='https://treeherder.mozilla.org')
```

Comment:
Yeah, I can remove this because for fetching build data from Treeherder will always have to use the production instance.